model/src/lib.rs

```diff
@@ -62,15 +62,14 @@ pub enum Error {
     NegativePrice,
 }
 
-/// Represents "quantity" or "contract size" in Cfd terms
 #[derive(Debug, Copy, Clone, PartialEq, PartialOrd, Serialize, Deserialize)]
-pub struct Usd(Decimal);
+pub struct Contracts(Decimal);
```

Comment:
🤔 Do we want to model contracts using `Decimal` instead of non-negative integers, like `u64`?

I would assume that fractional contracts make no sense.

Reply:
I opted for decimal for simpler calculations. Might be a bit more work to change to `u64`. What do you think?

Reply:
I think it's important for our core types to be correct, so I would opt for `u64`.

I would only convert to `Decimal` whenever we need to calculate things. We already have to do this for types backed by `Amount` or `SignedAmount`.

In any case, we don't have to tackle this now. Perhaps we can create an issue if you agree with my assessment.

Reply:
I'll create a ticket. I think we should then use checked math for contract arithmethic because of overflows etc.

Reply:
https://github.com/itchysats/itchysats/issues/2735